repository_module/src/main/java/ru/kazimir/bortnik/online_market/repository/UserRepository.java

```diff
@@ -0,0 +1,22 @@
+package ru.kazimir.bortnik.online_market.repository;
+
+import ru.kazimir.bortnik.online_market.repository.model.User;
+
+import java.sql.Connection;
+import java.util.List;
+
+public interface UserRepository extends GenericRepository {
+    List<User> getUsers(Connection connection, Long limitPositions, Long positions);
+
+    Long getSizeData(Connection connection);
```

Comment:
What is size data? need to rename to appropriate method name for example getCountOfUsers